assets/src/stories-editor/components/context-menu/index.js

```diff
@@ -153,9 +153,19 @@ const ContextMenu = ( props ) => {
 			return;
 		}
 
+		let parentBlock = rootClientId;
+		let index = null;
+
+		if ( block.name === 'amp/amp-story-page' ) {
```

Comment:
I still think you should use `isPage` here. Note: Not `isPageBlock` helper function, but `isPage` variable from outer scope.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3537/commits/592ea8907303218647949ff204a188d668b25263